src/sources/statsd/mod.rs

```diff
@@ -152,13 +154,13 @@ impl Deserializer for StatsdDeserializer {
             .and_then(parse)
         {
             Ok(metric) => {
-                emit!(&StatsdEventReceived {
+                emit!(&StatsdEventsReceived {
                     byte_size: bytes.len()
```

Comment:
Similar to other PR comments, I think the `byte_size` being used here is wrong. It should be using `ByteSizeOf::size_of` to emit a counter of in-memory bytes.